configure

```diff
@@ -1123,6 +1128,9 @@ for ac_option in $@; do
 	--disable-faad)               _faad=no               ;;
 	--enable-faad)                _faad=yes              ;;
 	--disable-fluidsynth)         _fluidsynth=no         ;;
+	--enable-fluidsynth)          _fluidsynth=yes        ;;
+	--disable-fluidlite)          _fluidlite=no          ;;
+	--enable-fluidlite)           _fluidlite=yes         ;;
```

Comment:
There is this in `gui/module.mk`

```
ifdef USE_FLUIDSYNTH
MODULE_OBJS += \
	fluidsynth-dialog.o
endif
```
Is that maybe needed for `fluidlite`, too?

Reply:
``USE_FLUIDSYNTH`` is defined when fluidlite is enabled, so that shouldn't be necessary.

Reply:
ah - missed the part where you also define USE_FLUIDSYNTH in config.mk - sorry.